src/utilities/files-helper.ts

```diff
@@ -68,6 +69,18 @@ export function writeManifestToFile(inputObjectString: string, kind: string, nam
     return '';
 }
 
+export function getManifestFileContents(filePath: string) {
+    if (filePath &&
+        filePath.length > 0) {
+        try {
+            return yaml.safeLoad(fs.readFileSync(filePath, { encoding: "utf-8" }));
+        } catch (ex) {
+            core.debug('Exception occurred while reading manifest file : ' + filePath + ' . Exception: ' + ex);

```

Comment:
Prefer string interpolation for readability:
`` `Exception occurred while reading manifest file: '${filePath}'. Exception: '${ex}'` ``